### PR TITLE
Add Dockerfile for Tower deploy

### PR DESCRIPTION
**Pull request by pmastalerz**

### CODE REVIEW

File: Dockerfile.production [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+ENV LANG en_US.UTF-8
+ENV LANGUAGE en_US:en
+ENV LC_ALL en_US.UTF-8
+ENV AWS_ACCESS_KEY ${AWS_ACCESS_KEY}
+ENV AWS_SECRET_ACCESS_KEY ${AWS_SECRET_ACCESS_KEY}
+
+RUN apt-get update && \
+    apt-get install -y openjdk-7-jre-headless && \
+    gem install s3_website scss_lint jekyll --no-rdoc --no-ri && \ 
+    npm install gulp http-server -g && \
```
Comment (aju) [inline Dockerfile.production]: This is not needed - please check

File: s3_website.yml [hunk 2]
```diff
@@ -1,7 +1,6 @@
 s3_id: <%= ENV['AWS_ACCESS_KEY'] %>
 s3_secret: <%= ENV['AWS_SECRET_ACCESS_KEY'] %>
-s3_bucket: brainly-styleguide
-cloudfront_distribution_id: EZLBBBYW4V6SO
+s3_bucket: styleguide.brainly.com
 
 # Below are examples of all the available configurations.
```
Comment (aju) [inline s3_website.yml]: cleanup usless comments

**Comment by aju** [comment]
Update current docker as it will not need s3_website gem anymore

**Comment by aju** [comment]
OK

**Comment by vergilius** [comment]
LGTM

**Comment by aju** [comment]
LGTM

**Pull request merged**